Scoop/Controller/Onboarding/OnboardingViewController.swift

```diff
@@ -32,11 +34,83 @@ class OnboardingViewController: UIViewController {
         view.addSubview(button)
         
         button.snp.makeConstraints { make in
-
             let screenSize = UIScreen.main.bounds
-            print(screenSize.width)
-            make.trailing.equalTo(view.safeAreaLayoutGuide).inset(screenSize.width * trailingButtonMultiplier)
+            make.trailing.equalTo(view.safeAreaLayoutGuide).inset(20)
             make.bottom.equalTo(view.safeAreaLayoutGuide).inset(screenSize.height * bottomButtonMultiplier)
         }
     }
+    
+    func setupBackButton() {
+        backButton.setImage(UIImage(named: "BackArrow"), for: .normal)
+        backButton.addTarget(self, action: #selector(prevVC), for: .touchUpInside)
+        self.navigationController?.navigationBar.addSubview(backButton)
+        backButton.snp.makeConstraints { make in
+            make.leading.equalToSuperview().inset(25)
+            make.top.centerY.equalToSuperview()
```

Comment:
You shouldn't be centering the Y and position to the superview. That would result in conflicting constraints. Instead I would choose which one you want to actually pin to the superview